mu4e/mu4e-headers.el

```diff
@@ -1570,7 +1607,16 @@ searching. If SHOW is non-nil, show the message with MSGID."
           (if edit
               (read-string prompt expr)
             (or expr
-                (read-string prompt nil 'mu4e~headers-search-hist)))))
+                (if (fboundp 'minibuffer-with-setup-hook)
+                    ;; Available since Emacs 25.1.
+                    (minibuffer-with-setup-hook
```

Comment:
We require emacs 25.3, so no need for the check.

Reply:
Done!